src/main/resources/templates/index.html

```diff
@@ -3,59 +3,18 @@
 <head>
     <meta charset="UTF-8">
     <title>Market Data</title>
-    <style>
-        body{font-family: Consolas,serif;}
-        .panel{ width: 400px; height: 400px; border: solid black 1px}
-    </style>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
 </head>
+<body>
```

Comment:
Body tag was in the wrong place previously. Moving it here allows the file to pass validation.

Reply:
Good.